modules/ROOT/pages/_partials/include-manage-proxies.adoc

```diff
@@ -1,10 +1,10 @@
 // tag::monitoring-proxy[]
 == Configure an Anypoint Monitoring Proxy
 
-Because the monitoring agent requires a SOCKS5 proxy, you need two proxy types if you use Anypoint Monitoring.
 
-[NOTE]
-Starting Runtime Fabric version 2.6.22, the following HTTP proxy is needed for Anypoint Monitoring sidecar or agents as well. SOCK5 proxy is not required for Mule apps deployed on Runtime Fabrics with version 2.6.22 onwards.
+For Runtime Fabric deployments on version 2.6.22 and later, Anypoint Monitoring uses an HTTP proxy. Refer to <<configure-http, Configure an HTTP Proxy for Services in Runtime Fabric>> for details. 
```

Comment:
```suggestion
For Runtime Fabric deployments on version 2.6.22 and later, Anypoint Monitoring uses an HTTP proxy. Refer to <<configure-http>> for details. 
```
The title gets autopopulated in cases like this where it's above a section title.